specs/tree-view/tree-view.md

```diff
@@ -0,0 +1,195 @@
+# Tree View + Tree View Item
+
+## Overview
+
+There are technically two components being presented as part of this specification - tree view, and tree view item. 
+
+As defined by the W3C:
+> A tree view widget presents a hierarchical list. Any item in the hierarchy may have child items, and items that have children may be expanded or collapsed to show or hide the children. For example, in a file system navigator that uses a tree view to display folders and files, an item representing a folder can be expanded to reveal the contents of the folder, which may be files, folders, or both.
+
+### Background
+
+We recently created a React implementation of Tree View and Tree View Item with [PR #2510](https://github.com/microsoft/fast-dna/pull/2510). 
```

Comment:
I don't think this line belongs in a spec.

Reply:
I think that’s fair, it simply gives context to the feature request which follows which I think is helpful in informing the control decisions.